app/controllers/identified_clients_controller.rb

```diff
@@ -67,42 +61,53 @@ def client_source
 
     def identified_client_params
       params.require(:identified_client).permit(
-        :client_identifier,
-        :assessment_score,
         :agency_id,
         :contact_id,
         :first_name,
         :last_name,
         :middle_name,
         :date_of_birth,
         :ssn,
-        :days_homeless_in_the_last_three_years,
-        :date_days_homeless_verified,
-        :who_verified_days_homeless,
-        :veteran,
-        :rrh_desired,
-        :youth_rrh_desired,
-        :rrh_assessment_contact_info,
-        :income_maximization_assistance_requested,
-        :pending_subsidized_housing_placement,
         :full_release_on_file,
-        :requires_wheelchair_accessibility,
-        :required_number_of_bedrooms,
-        :required_minimum_occupancy,
-        :requires_elevator_access,
-        :family_member,
-        :calculated_chronic_homelessness,
-        :gender,
         :available,
-        :income_total_monthly,
-        :disabling_condition,
-        :physical_disability,
-        :developmental_disability,
-        :domestic_violence,
-        :interested_in_set_asides,
-        :actively_homeless,
-        :active_cohort_ids => [],
-        :neighborhood_interests => [],
+        :active_client,
+        :eligible_for_matching,
+        active_cohort_ids: [],
+        client_assessments_attributes: [
+          :id,
+          :type,
+          :assessment_score,
+          :actively_homeless,
+          :days_homeless_in_the_last_three_years,
+          :date_days_homeless_verified,
+          :who_verified_days_homeless,
+          :veteran,
+          :rrh_desired,
+          :youth_rrh_desired,
+          :rrh_assessment_contact_info,
+          :income_maximization_assistance_requested,
+          :pending_subsidized_housing_placement,
+          :requires_wheelchair_accessibility,
+          :required_number_of_bedrooms,
+          :required_minimum_occupancy,
+          :requires_elevator_access,
+          :family_member,
+          :calculated_chronic_homelessness,
+          :income_total_annual,
+          :disabling_condition,
+          :physical_disability,
+          :developmental_disability,
+          :domestic_violence,
+          :interested_in_set_asides,
+          :youth_rrh_aggregate,
+          :dv_rrh_aggregate,
+          :veteran_rrh_desired,
+          :rrh_th_desired,
+          :sro_ok,
+          :other_accessibility,
+          :disabled_housing,
+          neighborhood_interests: [],
```

Comment:
This looks very much the same as the deidentified version.  We should probably move it to the model to DRY it up a bit.